src/corner_kick/.yarnrc

```diff
@@ -0,0 +1,2 @@
+# Enforce a specific yarn version
+yarn-path "./scripts/yarn-1.13.0.js"
```

Comment:
GIthub complaining about lack of newline

Reply:
Fixed